openfeature-provider/java/src/test/java/com/spotify/confidence/ResolveTest.java

```diff
@@ -292,8 +360,59 @@ private static byte[] buildResolverStateBytes(Map<String, Flag> flagsMap) {
     builder.addClientCredentials(
         com.spotify.confidence.iam.v1.ClientCredential.newBuilder()
             .setName(credentialName)
-            .setClientSecret(TestBase.secret)
+            .setClientSecret(secret)
             .build());
     return builder.build().toByteArray();
   }
+
+  private ResolveFlagsResponse resolveWithContext(
+      List<String> flags, String username, Struct struct, boolean apply, String secret) {
+    return resolverApi
+        .resolveWithSticky(
```

Comment:
nit: I suggest we have a single `resolve` signature with optional stickiness in the request. Not for this PR